src/main/java/uk/gov/hmcts/darts/event/service/handler/DarStartHandler.java

```diff
@@ -30,8 +31,8 @@ public DarStartHandler(RetrieveCoreObjectService retrieveCoreObjectService,
     @Override
     @Transactional
     public void handle(DartsEvent dartsEvent, EventHandlerEntity eventHandler) {
-        createHearingAndSaveEvent(dartsEvent, eventHandler); // saveEvent
-        var notifyEvent = new DarNotifyApplicationEvent(this, dartsEvent, START_RECORDING);
+        CreatedHearingAndEvent hearingAndSaveEvent = createHearingAndSaveEvent(dartsEvent, eventHandler);// saveEvent
+        var notifyEvent = new DarNotifyApplicationEvent(this, dartsEvent, START_RECORDING, hearingAndSaveEvent.getHearingEntity().getCourtroom().getId());
```

Comment:
minor:
I wonder whether it would be better to pass the whole `HearingEntity` to `DarNotifyApplicationEvent` rather than the courtRoomId, so that if in the future we need to access aslo the hearing id we don't have to change the `DarNotifyApplicationEvent` signature again